### PR TITLE
profile tag added

### PR DESCRIPTION
**Pull request by ursmahi**
This PR fixes open issue #1301

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots
screenshot

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2204"></a>

### CODE REVIEW

File: pages/[username].js [hunk 1]
```diff
@@ -73,6 +73,16 @@ export default function User({ data }) {
         <div className="flex justify-center my-4">
           <ReactMarkdown>{data.bio}</ReactMarkdown>
         </div>
+        <div className="flex justify-center">
+          {
+            data?.tags?.length > 0 && 
+            data.tags.map((tag, index) => (
```
Comment (eddiejaoude) [inline pages/[username].js:79]: I think it might be easier to read with something like...

```js
{ data.tags && data.tags.map(
```

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Looks great 👍 

1 small inline comment, and please could you add it to the docs in the README that peel can use "tags" now in their data

File: pages/[username].js (hunk 1 above)
Comment (ursmahi) [reply pages/[username].js:79]: yes, but as no user have tags now in their profile we will get error
we can try this `{ data?.tags && data.tags.map(`

**Comment by eddiejaoude** [comment]
@ursmahi why did you close it? You can add more commits to the same PR

**Comment by ursmahi** [comment]
> @ursmahi why did you close it? You can add more commits to the same PR

This is my first PR, I am confused

File: pages/[username].js (hunk 1 above)
Comment (eddiejaoude) [reply pages/[username].js:79]: `data` is always set, so there will not be an error

**Comment by eddiejaoude** [comment]
> This is my first PR, I am confused

No problem, try things out and any questions you can ask, it is all fixable on GitHub, so don't worry

Btw you can ask questions in the EddieHub Discord, everyone is super friendly and help in there http://discord.eddiehub.org

**Comment by ursmahi** [comment]
I have updated the change requests, please review it

**Comment by ursmahi** [comment]
> Almost there, please see this comment https://github.com/EddieHubCommunity/LinkFree/pull/2204/files#r1020279323
> 
> Note: we should add tests but this can come in another PR

updated the changes please check

**Review by eddiejaoude — APPROVED** [review]
Looks great 👍 thank you

**Pull request merged**